### PR TITLE
Add some documentation on the multi cluster setup

### LINKED ISSUE #355: doc: Multicluster documentation
Write some documentation about the multi cluster setup using Razee.

### PR DESCRIPTION
**Pull request by froesef**
Fixes #355

### CODE REVIEW

File: website/content/docs/setup/multicluster/index.md [hunk 1]
```diff
@@ -1,8 +1,71 @@
...
+### Installing using IBM Satellite Config
+
+When using [IBM Satellite Config](https://cloud.ibm.com/satellite) the RazeeDash API is running as a service in the
+cloud and all CRD distribution is handled by the cloud. The process here describes how an already existing Kubernetes
+cluster can be registered and configured.
+
+![IBM Satellite config](ibm_satellite_conf_cluster.png)
+
+1. In the IBM Satellite Cloud service under the `Clusters` tab click on `Attach cluster`.
+2. Enter a cluster name in the popup dialog and click `Register cluster`. (Please don't use spaces in the name)
```
Comment (roee88) [inline website/content/docs/setup/multicluster/index.md:51]: style: use **bold** for UI elements

**Review by roee88 — COMMENTED** [review]
It feels like with some minor work on the Helm chart we can reduce the text here significantly. Would you like to discuss and help with that?

**Comment by froesef** [comment]
> It feels like with some minor work on the Helm chart we can reduce the text here significantly. Would you like to discuss and help with that?

I completely agree! This should be part of the helm chart! I'm happy to contribute that part. I'll work on that once the other PRs for the helm chart are merged.

File: website/content/docs/setup/multicluster/index.md [hunk 2]
```diff
@@ -1,8 +1,71 @@
...
+the deployment status of such deployments. There are multiple ways to run Razee. The two desribed here are a vanilla open source deployment on your
+own Kubernetes or as a managed service from a cloud provider. The two different setups are described below.
+In general there is a need for:
+* Razee watch keeper (installed on all clusters)
+* Razee cluster subscription manager (installed on all clusters)
+* RazeeDash API (installed on controller cluster/as cloud service)
+
+### Installing Razee on Kubernetes
+
+An installation of the open source components is described [here](https://razee.io/#get-razee).
```
Comment (ronenkat) [inline website/content/docs/setup/multicluster/index.md:24]: are there non open source components that are to be installed?

File: website/content/docs/setup/multicluster/index.md [hunk 3]
```diff
@@ -1,8 +1,71 @@
...
+* Razee watch keeper (installed on all clusters)
+* Razee cluster subscription manager (installed on all clusters)
+* RazeeDash API (installed on controller cluster/as cloud service)
+
+### Installing Razee on Kubernetes
+
+An installation of the open source components is described [here](https://razee.io/#get-razee). 
+(TODO maybe add a description on how to install it including openshift routes etc)
+
+The credentials to access Razee have to be configured in a special secret in the `m4d-system` namespace.
```
Comment (ronenkat) [inline website/content/docs/setup/multicluster/index.md:27]: do you want to expand on "special secret"?

File: website/content/docs/setup/multicluster/index.md [hunk 4]
```diff
@@ -1,8 +1,71 @@
...
+
+```
+envFrom:
+- secretRef:
+    name: razee-credentials
+- configMapRef:
+    name: m4d-config
+```
+
+### Installing using IBM Satellite Config
```
Comment (ronenkat) [inline website/content/docs/setup/multicluster/index.md:42]: Is this an alternative way to install Razee on a cluster? if so, can we emphasize, and add the context which is that IBM Cloud offers a aaS way to use Razee, and include prerequisite (i.e., at least need an IBM cloud account, use IAM keys, ...).

**Review by ronenkat — COMMENTED** [review]
@froesef Thank you.
Looks good, added feedback about readability.

**Pull request closed without merging**